apps/communications/dialer/test/unit/call_log_test.js

```diff
@@ -790,9 +790,19 @@ suite('dialer/call_log', function() {
       sinon.assert.notCalled(CallLog.renderEmptyCallLog);
     });
 
-    test('renderEmptyCallLog should be called when there is no missedgroup',
+    test('renderEmptyCallLog should be called when there is no missedgroup ' +
+         'and it is in missed view',
     function() {
       this.sinon.spy(CallLog, 'renderEmptyCallLog');
+      CallLog.appendGroup(missedGroup);
+      CallLog.filter();
+      var container = CallLog.callLogContainer;
+      var checkboxes =
+        container.querySelectorAll('.missed-call input[type="checkbox"]');
+      for (var i = 0; i < checkboxes.length; i++) {
+        var checkbox = checkboxes[i];
+        checkbox.checked = true;
+      }
       CallLog.deleteLogGroups();
       ConfirmDialog.executeYes();
       sinon.assert.calledOnce(CallLog.renderEmptyCallLog);
```

Comment:
Would it be possible to change this test a bit so that it verifies that in addition to just the renderEmptyCallLog being called that the correct message for the noMissedCallsMsg is displayed in the right filter tab?  Otherwise, I think the test passes both with and without the change to call_log.js.